package.json

```diff
@@ -44,6 +44,8 @@
     "@react-router/express": "7.4.1",
     "@react-router/fs-routes": "7.4.1",
     "@react-router/node": "7.4.1",
+    "@scure/bip32": "1.6.2",
+    "@scure/bip39": "1.5.4",
```

Comment:
can you pls check how much did these libs increase our bundle size? (use some bundle analyser or run the build script and check the sizes of generated files before and after this lib)

Reply:
it's fine. 

> Should we use the same versions that cashu-crypto uses?

no this is fine. cashu-crypto defines those deps like this `"@scure/bip39": "^1.4.0"`, "@scure/bip32": "^1.5.0" and versions you picked are compatible with that so we should end up with one version.

resolving